TravelPoints-BackEnd/src/main/java/com/example/travelpointsbackend/controller/ReviewController.java

```diff
@@ -0,0 +1,74 @@
+package com.example.travelpointsbackend.controller;
+
+import com.example.travelpointsbackend.dto.ReviewDTO;
+import com.example.travelpointsbackend.model.Review;
+import com.example.travelpointsbackend.service.ReviewService;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+@CrossOrigin(origins = "http://localhost:4200")
+@RestController
+@RequestMapping("/reviews")
+@RequiredArgsConstructor
+public class ReviewController {
+    private static final String ADD_FAILED = "{\"message\": \"Add review failed!\"}";
+    private static final String DELETE_SUCCEED = "{\"message\": \"Delete review succeed!\"}";
+    private static final String DELETE_FAILED = "{\"message\": \"Delete review failed!\"}";
+    private static final String UPDATE_SUCCEED = "{\"message\": \"Update review succeed!\"}";
+    private static final String UPDATE_FAILED = "{\"message\": \"Update review failed!\"}";
+    private static final String GET_FAILED = "{\"message\": \"Get reviews failed!\"}";
```

Comment:
C: These constants are pointless unless you want to send a more detailed message back to the frontend. If a request succeded or failed should be determined from the status code